src/components/UnbindORCiD.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { SynapseClient, Typography } from 'synapse-react-client'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { PROVIDERS } from 'synapse-react-client/dist/containers/Login'
+import { ValidationWizardStep } from './ProfileValidation'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import { Button, Modal } from 'react-bootstrap'
+
+
+export const unbindORCiD = async (orcid: string|undefined, accessToken:string|undefined, redirectAfter?:string) => {
+    if(orcid){
+        try {
+            await SynapseClient.unbindOAuthProviderToAccount(PROVIDERS.ORCID, accessToken, orcid)
+            if(redirectAfter){
```

Comment:
The following looks like code take from ORCiDButton.onBindToORCiD().  Please refactor to use the same function to bind a new ORCiD.